Dixit_Logic/Tests/DeckTest.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Dixit_Logic.Classes;
+using Dixit_Logic.Interfaces;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dixit_Logic.Tests
+{
+    [TestClass]
+    public class DeckTest
+    {
+        [TestMethod]
+        public void TestAddCardAndAddCardsAndEvacuateMethods()
+        {
+            Deck firstDeck = new Deck();
+            Deck secondDeck = new Deck();
+
+            List<ICard> cardList = new List<ICard>();
+
+            for (int i = 0; i < 10; i++)
+            {
+                Card newCard = new Card(i);
+                firstDeck.AddCard(newCard);
+                cardList.Add(newCard);
```

Comment:
i add the newCard to the cardList too, becous in the next lines the program construates a secondDeck based on this list.